heron/common/src/python/handler/access/BUILD

```diff
@@ -0,0 +1,9 @@
+package(default_visibility = ["//visibility:public"])
+
+load("/tools/rules/pex_rules", "pex_library")
+
+pex_library(
+  name = 'tracker-access',
+  srcs = glob(['**/*.py']),
+  deps = [],
+)
```

Comment:
Instead of creating a new BUILD file, please add a new rule in the existing common/src/python/BUILD file.

Please refer the BUILD file for spi: https://github.com/twitter/heron/blob/master/heron/spi/src/java/BUILD#L163